src/MSBuild/app.amd64.config

```diff
@@ -190,13 +190,13 @@
         </dependentAssembly>
         <dependentAssembly>
           <assemblyIdentity name="System.Memory" publicKeyToken="cc7b13ffcd2ddd51" culture="neutral" />
-          <bindingRedirect oldVersion="0.0.0.0-4.0.1.2" newVersion="4.0.1.2" />
-          <codeBase version="4.0.1.2" href="..\System.Memory.dll"/>
+          <bindingRedirect oldVersion="0.0.0.0-4.0.1.2" newVersion="4.0.2.0" />
```

Comment:
Is there any particular reason you deviate from the norm I see in this file and elsewhere, where the high bound of the old version is equal to the new version? i.e. Do you mean to exclude 4.0.1.3 from the redirect?

Reply:
No, nice catch!

Reply:
```suggestion
          <bindingRedirect oldVersion="0.0.0.0-4.0.2.0" newVersion="4.0.2.0" />
```